### PR TITLE
FEAT(grades-route): implemented first version of grades route view

### PR DESCRIPTION
**Pull request by mvoemel**
image

### CODE REVIEW

File: client/react/src/components/grades/grades-view.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+const GradesView = () => {
+  return (
+    <Card className="font-mono text-sm min-w-full max-w-5xl mx-auto p-4 rounded-lg">
+      <div className="grid grid-cols-2 mb-2 px-4">
+        <span>Name</span>
+        <span className="text-right">ECTS & Grade</span>
+      </div>
+      <div className="space-y-2">
+        {degrees.map((degree, index) => {
+          // if is last return without separator
```
Comment (kryezleo) [inline client/react/src/components/grades/grades-view.tsx:22]: What do you think about the idea to outsource the logic in a separate file to enhance the readability?

**Review by kryezleo — APPROVED** [review]

File: client/react/src/components/grades/grades-view.tsx (hunk 1 above)
Comment (mvoemel) [reply client/react/src/components/grades/grades-view.tsx:22]: the degrees.map logic i think we can leave but the other components (DegreeSection, SemesterSection, ...) should be outsourced in an upcoming refactor

Comment (mormeli) [reply client/react/src/components/grades/grades-view.tsx:22]: I agree with @mvoemel.

**Review by mormeli — COMMENTED** [review]
Although my knowledge of React is limited, the code looks good to me so far.

**Comment by ShpetimV** [comment]
Looks good to me.

**Review by ShpetimV — APPROVED** [review]
ready to merge into dev.

**Pull request merged**